src/modules/languages/php.nix

```diff
@@ -248,19 +291,25 @@ in
     };
   };
 
-  config = lib.mkIf cfg.enable {
-    packages = with pkgs; [
-      cfg.package
-    ] ++ lib.optional (cfg.packages.composer != null) cfg.packages.composer;
+  config =
+    let
+      customPhpPackage = (if ((builtins.hasAttr "php${version}" pkgs) && (builtins.tryEval (toString pkgs."php${version}")).success) then pkgs."php${version}" else phps.packages.${pkgs.system}."php${version}");
```

Comment:
- the outer parenthesis isn't needed
- it would be good to split up the conditionals in multiple lines for readability
- if `phps.packages.${system}.php${version} isn't there, we should `throw` an error to the list of supported versions.

Reply:
![image](https://user-images.githubusercontent.com/6224096/217011910-a6a916f1-5912-4889-a4b0-25ce69ec146a.png)
done :)